wbooks/src/redux/book/reducer.ts

```diff
@@ -0,0 +1,25 @@
+import { BOOKS } from '@constants/types';
+
+import { actionCreators } from './actions';
```

Comment:
This is not correct, the actionCreators must be called from another actions or components/screens. Never put logic in the reducer, it is only for state managment